File: cote/kakao_210508/ka1.py
# 카카오 인턴십 1

import sys
from collections import deque
import copy
rd = sys.stdin.readline

def solution(s):
    words = {"zero": "0",
             "one": "1",
             "two": "2",
             "three": "3",
             "four": "4",
             "five": "5",
             "six": "6",
             "seven": "7",
             "eight": "8",
             "nine": "9"}

    answer = ""
    tempStr = ""
    for char in s:
        if char > "9":
            tempStr += char
        else:
            answer += char
        
        if  tempStr in words:
            answer += words[tempStr]
            tempStr = ""
    
    return int(answer)


print(solution("one4seveneight"))
print(solution("23four5six7"))

File: cote/kakao_210508/ka1.swift
// 2021 카카오 인턴십 1

import Foundation

func solution(_ s:String) -> Int {
  let words = ["zero": "0",
               "one": "1",
               "two": "2",
               "three": "3",
               "four": "4",
               "five": "5",
               "six": "6",
               "seven": "7",
               "eight": "8",
               "nine": "9"]

  var str = ""
  var answer = ""
  for char in s {
    if char > "9" {
      str += String(char)
    } else {
      answer += String(char)
    }

    if let temp = words[str] {
      answer += temp
      str = ""
    }
  }
  return Int(answer)!
}

print(solution("one4seveneight"))
print(solution("23four5six7"))